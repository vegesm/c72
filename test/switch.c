main () {
    int a;
    a=0;
    switch(a) {
        case 0:
            printf("test1 - 0\n");
        case 1:
            printf("test1 - 1\n");
        default:
            printf("test1 - default\n");
    }

    a=1;
    switch(a) {
        case 0:
            printf("test2 - 0\n");
            break;
        case 1:
            printf("test2 - 1\n");
            break;
        default:
            printf("test2 - default\n");
            break;
    }

    a=5;
    switch(a) {
        case 0:
            printf("test3 - 0\n");
        case 1:
            printf("test3 - 1\n");
        default:
            printf("test3 - default\n");
    }

    printf("end of program\n");
}