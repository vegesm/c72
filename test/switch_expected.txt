test1 - 0
test1 - 1
test1 - default
test2 - 1
test3 - default
end of program
